Add option to disable vsphere csi (#3664)
aws/eks-anywhere: docs/content/en/docs/reference/clusterspec/vsphere.md

If you specify the wrong thumbprint, an error message will be printed with the expected thumbprint. If no valid
certificate is being used, `insecure` must be set to true.

### disableCSI (optional)
Set `disableCSI` to `true` if you don't want to have EKS Anywhere install and manage the vSphere CSI driver for you. 
More details on the driver are [here](https://docs.vmware.com/en/VMware-vSphere-Container-Storage-Plug-in/2.0/vmware-vsphere-csp-getting-started/GUID-C44D8071-85E7-4933-83EA-6797518C1837.html)

> >**_NOTE:_** If you upgrade a cluster and disable the Vvphere CSI driver after it has already been installed by EKS Anywhere, 

Reviewer: ```suggestion
>**_NOTE:_** If you upgrade a cluster and disable the vSphere CSI driver after it has already been installed by EKS Anywhere, 
```

If you specify the wrong thumbprint, an error message will be printed with the expected thumbprint. If no valid
certificate is being used, `insecure` must be set to true.

### disableCSI (optional)

> Set `disableCSI` to `true` if you don't want to have EKS-A install and manage the VSphere CSI driver for you. 

Reviewer: ```suggestion
Set `disableCSI` to `true` if you don't want to have EKS Anywhere install and manage the vSphere CSI driver for you. 
```

> More details on the driver is [here](https://docs.vmware.com/en/VMware-vSphere-Container-Storage-Plug-in/2.0/vmware-vsphere-csp-getting-started/GUID-C44D8071-85E7-4933-83EA-6797518C1837.html)

Reviewer: ```suggestion
More details on the driver are [here](https://docs.vmware.com/en/VMware-vSphere-Container-Storage-Plug-in/2.0/vmware-vsphere-csp-getting-started/GUID-C44D8071-85E7-4933-83EA-6797518C1837.html)
```

> >**_NOTE:_** If you upgrade a cluster and disable the VSphere CSI driver after it has already been installed by EKS-A, 

Reviewer: ```suggestion
>**_NOTE:_** If you upgrade a cluster and disable the vSphere CSI driver after it has already been installed by EKS Anywhere, 
```

> > you will need to remove the resources manually from the cluster. Delete the `DaemonSet` and `Deployment` first as they 

Reviewer: ```suggestion
> you will need to remove the resources manually from the cluster. Delete the `DaemonSet` and `Deployment` first, as they 
```

> rely on the other resources.
> 

> > There are the following resources you would need to delete:

Reviewer: ```suggestion
> These are the resources you would need to delete:
```

> * vsphere-csi-controller-role (kind: ClusterRole)
> * vsphere-csi-controller-binding (kind: ClusterRoleBinding)
> * csi.vsphere.vmware.com (kind: CSIDriver)
> 

> > These are the following resources you would need to delete

Reviewer: ```suggestion
> These are the resources you would need to delete
```
